gopro

```diff
@@ -58,6 +61,20 @@ function run_args {
     >&2 echo " * Non-interactive:  $GOPRO_NON_INTERACTIVE"
     >&2 echo " * Autostart:        $GOPRO_AUTOSTART"
     >&2 echo " * Preview:          $GOPRO_PREVIEW"
+    >&2 echo " * Resolution:       $RES"
+    fov=""
+    case $FOV in
+  	  0)
+  	   fov="wide"
+  	   ;;
+  	  4)
+  	   fov="linear"
+  	   ;;
+  	  "6")
```

Comment:
Is there a reason 6 is quoted, but 0 and 4 are not?

Reply:
No reason, i teste 6 without quoted and works!